38-Count_and_Say.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+'''
+The count-and-say sequence is the sequence of integers with the first five terms
+as following:
+
+1.     1
+2.     11
+3.     21
+4.     1211
+5.     111221
+1 is read off as "one 1" or 11.
+11 is read off as "two 1s" or 21.
+21 is read off as "one 2, then one 1" or 1211.
+
+Given an integer n where 1 ≤ n ≤ 30, generate the nth term of the count-and-say
+sequence.
+
+Note: Each term of the sequence of integers will be represented as a string.
+
+
+
+Example 1:
+
+Input: 1
+Output: "1"
+Example 2:
+
+Input: 4
+Output: "1211"
+'''
+
+class Solution:
+    def countAndSay(self, n: 'int') -> 'str':
+        s = '1'
+        for _ in range(n - 1):
+            s = self.say(s)
+        return s
+
+
+    def say(self, sequence):
+        res = ''
+        count = 1
+        sequence += "0"
+        for i in range(len(sequence) - 1):
+            current = sequence[i]
+            next = sequence[i + 1]
```

Comment:
Avoid overwrite reserved keywords, such as "next".
https://www.programiz.com/python-programming/methods/built-in/next